app/scenes/Logout.tsx

```diff
@@ -1,10 +1,14 @@
 import * as React from "react";
 import { Redirect } from "react-router-dom";
+import env from "@shared/env";
 import useStores from "~/hooks/useStores";
 
 const Logout = () => {
   const { auth } = useStores();
   void auth.logout();
+  if (env.OIDC_LOGOUT_URI) {
+    window.location.replace(env.OIDC_LOGOUT_URI);
+  }
```

Comment:
Probably wise to `return null;` here to avoid a competing redirect

Reply:
Good point, added that